Dockerfile.slim

```diff
@@ -1,6 +1,6 @@
 FROM node:6-slim
 
-RUN groupadd teleirc -g 99999 ; useradd -u 99999 -g teleirc -s /bin/bash -M -d /opt/teleirc teleirc
+RUN groupadd teleirc -g 65533 ; useradd -u 65533 -g teleirc -s /bin/bash -M -d /opt/teleirc teleirc
```

Comment:
@robbyoconnor As a reference, could you explain why you changed the group and user IDs? I saw the commit e7396255ede70e83f8334cb5f0c8f3cc39a438e9 but wasn't sure why this needed to be done.

Reply:
@jflory7 65533 is the largest group ID that is available in the linux kernel.

Reply:
Yeh that was my screw up -- and I own it.

Reply:
Okay, cool, thanks for the clarification. 😁 Makes sense then!